### PR TITLE
Add Downloads page update to changelog

### PR DESCRIPTION
**Pull request by rileyhgrant**
Related: https://github.com/broadinstitute/gnomad-browser/pull/1069

[Preview link](https://gnomad.broadinstitute.org/news/preview/108/changelog)
[Preview link](https://gnomad.broadinstitute.org/news/preview/108/2023-02-updated-downloads-page/) (Links directly to the changelog entry)

Adds a changelog entry for the update to the Downloads page. To be merged/pushed to production when the corresponding Browser PR gets merged.

### CODE REVIEW

File: content/changelog/2023-02-updated-downloads-page.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Update downloads page
+date: "2023-02-02"
+order: 1
+---
+
+The [downloads page](https://gnomad.broadinstitute.org/downloads)'s format has been updated. The set of tabs that previously separated individual release's download links has been replaced by a sticky sidebar with a table of contents. Use the sidebar to to keep track of your position on the page, see at a glance what datasets are available for download, and quickly scroll the page to a dataset by clicking on it.
```
Comment (ch-kr) [inline content/changelog/2023-02-updated-downloads-page.md:7]: "see what datasets are available for download at a glance" reads a bit better, but I'm not sure it conveys the same message. maybe "see what datasets are available for download" is enough here, though -- what do you think?

**Review by ch-kr — APPROVED** [review]
minor rewording comment but LGTM!

**Review by mattsolo1 — CHANGES_REQUESTED** [review]
Going to change a bit of the wording here, 1 sec.

**Comment by mattsolo1** [comment]
We have redesigned the gnomAD [Downloads Page](https://gnomad.broadinstitute.org/downloads) to make it easier to explore datasets released by the gnomAD production team and gnomAD-affiliated research groups. 

Previously, our major releases (gnomAD v2, gnomAD v3, and ExAC) were organized under tabs that users would click to see which datasets were available for each release. This tab-based navigation scheme made it difficult to quickly understand the information hierarchy on the page. It also prevented users from searching the page using the standard internet browser search functionality (CTRL + F).

Now, instead of tabs, the new layout features a table of contents on the righthand side. Users will have a better sense of what is available and can quickly jump to specific releases and datasets. To help maintain a sense of context, the table of contents synchronizes with the page position as the user scrolls. Since all text is on the same page, CTRL-F searching is now possible.

Furthermore, the new Downloads Page attempts to distinguish which datasets are produced by the gnomAD production team ("Core Dataset") from downstream analyses carried out by affiliated research groups ("Secondary Analyses"). For example, our recently released [Genomic Constraint](https://gnomad.broadinstitute.org/downloads#research-genomic-constraint) analysis falls into the latter category.

**Comment by mattsolo1** [comment]
Btw if anyone feels my edit is too wordy... feel free to discard. Don't let this hold you up.

**Review by mattsolo1 — APPROVED** [review]
